### PR TITLE
Management authentication

### PR DESCRIPTION
**Pull request by solocommand**
Adds basic bootstrap wrapper, apollo client, and simple auth to the mangement service.
- [x] Log in
- [x] Log out
- [x] Sign up
- [x] Use litera
- [x] Final rebase
- [x] Invite user to org
- [x] Revoke/cancel invitation

screencapture-0-0-0-0-12002-login-2019-05-30-11_53_19
screencapture-0-0-0-0-12002-signup-2019-05-30-11_53_29

screencapture-0-0-0-0-12002-5cf6a934a50e560008ab6e7e-users-2019-06-04-13_48_04
screencapture-0-0-0-0-12002-5cf6a934a50e560008ab6e7e-users-2019-06-04-13_48_43
Screen Shot 2019-06-04 at 1 48 33 PM

### CODE REVIEW

File: services/manage/app/styles/bootstrap/_components.scss [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+@import "node_modules/bootstrap/scss/modal";
+@import "node_modules/bootstrap/scss/tooltip";
+@import "node_modules/bootstrap/scss/popover";
+@import "node_modules/bootstrap/scss/carousel";
+@import "node_modules/bootstrap/scss/spinners";
+@import "node_modules/bootstrap/scss/utilities";
+@import "node_modules/bootstrap/scss/print";
+
+// Remove outline and add pointer to all buttons
+button {
```
Comment (zarathustra323) [inline services/manage/app/styles/bootstrap/_components.scss]: Remove all these

File: services/manage/app/styles/bootstrap/_mixins.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
+@import "node_modules/bootstrap/scss/mixins";
+
+
+// Add custom mixins here...
+
+@mixin bg-gradient-variant($parent, $color) {
```
Comment (zarathustra323) [inline services/manage/app/styles/bootstrap/_mixins.scss]: Remove all these

File: services/manage/app/services/loading-display.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import Service from '@ember/service';
+import jQuery from 'jquery';
+
+export default Service.extend({
+  isShowing: false,
+  show() {
+    if (!this.get('isShowing')) {
+      this.set('isShowing', true);
+      jQuery('body').addClass('transitioning');
```
Comment (zarathustra323) [inline services/manage/app/services/loading-display.js]: Update this to use vanilla js

File: services/manage/app/services/graph-errors.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  handleNetworkError(networkError) {
+    const { response, result } = networkError;
+    const message = result.errors && Array.isArray(result.errors) ? result.errors[0].message : 'Unknown fatal.';
+    return new Error(`Network Error: ${response.statusText} (${response.status}): ${message}`);
+  },
+
+  show(e) {
+    const error = this.handle(e);
+    if (!e.loggingOut) {
+      // this.get('notify').error(error.message, { closeAfter: null });
```
Comment (zarathustra323) [inline services/manage/app/services/graph-errors.js]: Remove this

File: services/manage/app/services/graph-errors.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
+import Service, { inject } from '@ember/service';
+import { isPresent } from '@ember/utils';
+import $ from 'jquery';
+
+export default Service.extend({
+  // notify: inject(),
```
Comment (zarathustra323) [inline services/manage/app/services/graph-errors.js]: Remove this and all notify related items

**Comment by solocommand** [comment]
@zarathustra323 updated

**Review by zarathustra323 — APPROVED** [review]

**Pull request merged**